### PR TITLE
Output timestamp to utc

### PR DESCRIPTION
**Pull request by vunder-kind**

### CODE REVIEW

File: src/common/basic_analytics/basic_analytics_processor.py [hunk 1]
```diff
@@ -57,6 +58,8 @@ def create(self, read_stream):
         return [self.__convert_to_kafka_structure(result) for results in aggregated_results for result in results]
 
     def __convert_to_kafka_structure(self, dataframe):
+        utc = tz.gettz('UTC')
+        func = udf(lambda x: x.astimezone(utc).isoformat())
```
Comment (ltv12) [inline src/common/basic_analytics/basic_analytics_processor.py]: Please remove

File: src/common/basic_analytics/basic_analytics_processor.py [hunk 2]
```diff
@@ -57,6 +58,8 @@ def create(self, read_stream):
         return [self.__convert_to_kafka_structure(result) for results in aggregated_results for result in results]
 
     def __convert_to_kafka_structure(self, dataframe):
+        utc = tz.gettz('UTC')
```
Comment (ltv12) [inline src/common/basic_analytics/basic_analytics_processor.py]: Please remove unnecessary code

File: src/common/basic_analytics/basic_analytics_processor.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 from abc import ABCMeta, abstractmethod
 
+from dateutil.tz import tz
```
Comment (ltv12) [inline src/common/basic_analytics/basic_analytics_processor.py]: please remove unused import

File: src/common/basic_analytics/basic_analytics_processor.py (hunk 2 above)
Comment (vunder-kind) [reply src/common/basic_analytics/basic_analytics_processor.py]: Done

File: src/common/basic_analytics/basic_analytics_processor.py (hunk 3 above)
Comment (vunder-kind) [reply src/common/basic_analytics/basic_analytics_processor.py]: Done

**Pull request merged**